src/pages/en/guides/content.md

```diff
@@ -0,0 +1,75 @@
+
+You can use any Markdown editor, including your code editor, to compose your content.
+
+ Create a new `.md` file within your Astro repository, or add a new Markdown file written in an external editor. Some online Markdown editors like [StackEdit](https://stackedit.io/) or [Dillinger](https://dillinger.io) will even sync your work with your Astro repository stored on GitHub.
+
+📚 Learn more about [writing Markdown content in Astro](/en/guides/markdown-content/).
+
+### MDX Authoring
+With the optional Astro MDX integration added to your project, you can include JavaScript expressions and components within your Markdown content. Add UI elements such as a banner or an interactive carousel right in your text, to take your blogging to the next level!
+
+Write `.mdx` files directly in your code editor, alongside your project files. You can use frontmatter YAML, ESM import and export statements, and JavaScript expressions inside curly braces while writing your text content in standard Markdown.
+
+📚 Learn more about the [Astro official MDX integration](/en/guides/integrations-guide/mdx/).
+
+### Headless CMS Authoring
+
+Write blog posts in your existing Content Management System (CMS) such as Storyblok, WordPress, or Contentful. 
+
+Astro pages can [fetch your remote content](/en/guides/data-fetching/#fetch-from-a-headless-cms) using your service's API.
+
+## Adding new blog posts or pages
+
+Any supported page file (`.astro`, `.md`, `.mdx`, `.js`/.`ts`, `.html`) located within `src/pages/` automatically creates its own page route on your website with Astro's built-in [file-based routing](/en/core-concepts/routing/). Add a new post to your blog by adding or generating a new file anywhere within this directory! Individual pages will be built at a URL corresponding to the post's file path.
+
+If you are writing your content in a CMS, then you can take advantage of Astro's [dynamic routing](/en/core-concepts/routing/#dynamic-routes) by using a single Astro page component to create multiple blog posts and pages from your fetched content.
+
+## Adding Blog Infrastructure
+
+Once your content exists as pages on your site, you can use [Astro's API](/en/reference/api-reference/) and [integrations](/en/guides/integrations-guide/) to build standard blog features such as filtered or sorted lists of your posts, individual pages for tags or categories, previous and next navigation, and an RSS feed.
+
+These features require some one-time code set up. After that, your site will automatically update based on the current files in your project and any remote data returned by the most recent fetch calls. Note that for static websites, this updating occurs at build time only.
+
+:::tip[Official Blog Starter]
+Visit [astro.new](https://astro.new/) to see and play with existing code powering some of these features in our official Blog starter template!
```

Comment:
This is good placement for this, and now I especially want to remove the one in the page intro